### PR TITLE
Remove vtimer 

### PR DESCRIPTION
**Pull request by a3zzat**
cleanup the code to completely remove vtimer. 
to fix bug #4601

### CODE REVIEW

File: sys/net/routing/nhdp/iib_table.c [hunk 1]
```diff
@@ -118,7 +118,7 @@ iib_link_set_entry_t *iib_process_hello(kernel_pid_t if_pid, nib_entry_t *nb_elt
     }
 
     if (base_elt) {
-        vtimer_now(&now);
+      xtimer_now_timex(&now);
```
Comment (OlegHahm) [inline sys/net/routing/nhdp/iib_table.c]: indentation seems to be off here

File: sys/net/routing/nhdp/nhdp.c [hunk 2]
```diff
@@ -100,7 +101,10 @@ kernel_pid_t nhdp_start(void)
         /* Configure periodic timer message to refresh metric values */
         if (nhdp_pid != KERNEL_PID_UNDEF) {
             metric_interval = timex_from_uint64(DAT_REFRESH_INTERVAL * SEC_IN_USEC);
-            vtimer_set_msg(&metric_timer, metric_interval, nhdp_pid, NHDP_METRIC_TIMER, NULL);
+            metric_msg.type = NHDP_METRIC_TIMER;
+            metric_msg.content.ptr = NULL;
+            xtimer_set_msg64(&metric_timer, timex_uint64(metric_interval),
+                               metric_msg, nhdp_pid);
```
Comment (cgundogan) [inline sys/net/routing/nhdp/nhdp.c]: indentation is off here

**Comment by cgundogan** [comment]
Could you make a PR without the changes to `NHDP`? I am not very familiarized with the `NHDP` code and would like to review those changes separately

**Comment by OlegHahm** [comment]
@a3zzat, ping!

**Comment by cgundogan** [comment]
@a3zzat ping! Do you want me to adopt this PR if you have no time currently?

**Comment by kYc0o** [comment]
should we close this in favour of #5039 and #5040 ?

**Comment by gebart** [comment]
@kYc0o yeah, let's do that

**Pull request closed without merging**